File: two_groups/groups.py
def main():
    case_num = int(input())
    for _ in range(case_num):
        n = int(input())
        nums = list(map(int, input().split()))
        pos_sum = 0
        neg_sum = 0
        for num in nums:
            if num > 0:
                pos_sum += num
            elif num < 0:
                neg_sum += -num
        print(abs(pos_sum - neg_sum))


if __name__ == "__main__":
    main()

File: two_groups/groups.cpp
#include <iostream>

using namespace std;

int main()
{
    int case_num;
    cin >> case_num;
    for (int i = 0; i < case_num; i++)
    {
        int n;
        cin >> n;
        int num;
        long long pos_sum = 0;
        long long neg_sum = 0;
        for (int j = 0; j < n; j++)
        {
            cin >> num;
            if (num > 0)
            {
                pos_sum += num;
            }

            else if (num < 0)
            {
                neg_sum += -num;
            }
        }

        long long output = (pos_sum > neg_sum) ? pos_sum - neg_sum : neg_sum - pos_sum;
        cout << output << "\n";
    }
}
